package.json

```diff
@@ -1,7 +1,10 @@
 {
-  "name": "ember-easy-form-extensions",
+  "name": "@particle/ember-easy-form-extensions",
   "version": "1.0.14",
   "description": "Enhances Ember EasyForm by providing easy action handling, validations, and Ember 1.13 support for your forms",
+  "publishConfig": {
+    "access": "public"
```

Comment:
I'm not sure about the access restriction for this library.
IMO should be this one but I can change it if required @monkbroc